js/downloads.js

```diff
@@ -27,7 +27,7 @@ var packagesV11 = [hadoop3p2, hadoop2p7, hadoopFree, sources];
 
 
 addRelease("3.1.2", new Date("06/01/2021"), packagesV11, true);
-addRelease("3.0.2", new Date("02/19/2021"), packagesV10, true);
+addRelease("3.0.3", new Date("06/23/2021"), packagesV10, true);
```

Comment:
A side issue: this change is definitely correct. After 3.0.3 is released it should show up instead of 3.0.2, and 3.0.2 should be removed from dist.apache.org. I see that 3.0.2 was already removed: https://dist.apache.org/repos/dist/release/spark/ That is also the right thing to do, though I suppose really it should happen after we publish this web site change, or else the 3.0.2 link will redirect to mirrors that may be removing 3.0.2 from their mirror already.

No big deal as we're going to merge this soon and the download still worked for me now.

@Ngone51 if you have a second, and you went through the part of the release that removes old versions from the mirror, could you try removing 3.1.1 from https://dist.apache.org/repos/dist/release/spark/ ? it's replaced by 3.1.2 now.

I can definitely do that myself if that's any trouble.

Reply:
@srowen Thanks for the guidance. I'll pay attention next time. 

I can remove the 3.1.1 release:)

Reply:
Oh..I don't have permission to remove it. @dongjoon-hyun Could you help remove it since it's missed during the 3.1.2 release?

Reply:
Sure.

Reply:
Oh, it seems that 3.1.1 is already cleaned up.